nautobot/dcim/tables/devicetypes.py

```diff
@@ -253,7 +253,7 @@ class Meta(BaseTable.Meta):
             "name",
             "label",
             "type",
-            "power_port",
+            "power_port_template",
```

Comment:
Mind reviewing whether any of the other Template tables need similar corrections? For example, looking below at `FrontPortTemplateTable` I see that it has `rear_port` which should probably similarly be `rear_port_template`. That might be the only other one but it would be worthwhile to confirm.